<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parking</class>
 <widget class="QDialog" name="parking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>641</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="in_out_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>입출차내역</string>
   </property>
  </widget>
  <widget class="QLabel" name="in_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>현재 입차 내역</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
